packages/lit-dev-content/site/docs/ssr/authoring.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Authoring components for Lit SSR
+eleventyNavigation:
+  key: Authoring components
+  parent: Server rendering
+  order: 4
+---
+
+{% labs-disclaimer %}
+
+Lit's approach to rendering web components in a server environment places some restrictions on component code to achieve efficient server rendering. When authoring components, keep in mind these considerations if you your components to be compatible with Lit SSR.
+
+Note: The restrictions listed on this page are subject to change as we make improvements to Lit SSR. If you would like to see a certain use case supported, please [file an issue](https://github.com/lit/lit/issues/new/choose) or start a [discussion](https://github.com/lit/lit/discussions) thread.
+
+## Considerations
+
+### Browser only code
+
+Most browser DOM APIs are not available in the Node environment. Lit SSR utilizes a DOM shim that's the bare minimum required for rendering Lit templates and components. For a full list of what APIs are available, see the [DOM Emulation](/docs/ssr/dom-emulation) page.
+
+When authoring components, perform imperative DOM operations from lifecycle methods that are called only on the client, and not on the server. See the [lifecycles](#lifecycles) section below for what specific methods can be used.
+
+Some component modules may also have side effects that utilize browser APIs, to detect for certain browser features for example, which would break simply by importing in a non-browser environment. These side effects, if they cannot be moved to a client-only lifecycle, must have proper alternatives or some default behavior for when running in Node.
```

Comment:
```suggestion
Some component modules may also have side effects that utilize browser APIs, for example to detect certain browser features, which would break simply by importing in a non-browser environment. If these side-effects cannot be moved to a client-only lifecycle, you must provide proper alternatives or some default behavior for when running in Node.
```